packages/ra-core/src/auth/useCanAccess.spec.tsx

```diff
@@ -6,17 +6,10 @@ import { QueryClient } from '@tanstack/react-query';
 import { Basic } from './useCanAccess.stories';
 
 describe('useCanAccess', () => {
-    it('should return a loading state on mount', () => {
+    it('should allow access on mount when there is no authProvider', () => {
         render(<Basic authProvider={null} />);
-        expect(screen.queryByText('LOADING')).not.toBeNull();
-    });
-
-    it('should return isPending: true by default after a tick', async () => {
-        render(<Basic authProvider={null} />);
-        expect(screen.queryByText('LOADING')).not.toBeNull();
-        await waitFor(() => {
-            expect(screen.queryByText('LOADING')).toBeNull();
-        });
+        expect(screen.queryByText('LOADING')).toBeNull();
+        screen.getByText('canAccess: YES');
```

Comment:
You shouldn't replace these two tests, you should add one more

Reply:
Are we testing react-query?